### DISCUSSION TITLE
Make MultiViewer Open Source

### DISCUSSION THREAD

**Discussion by OmieSawie** [f1multiviewer/issue-tracker#561] [Feature requests]
Hey, I love Multiviewver, and I would love to contribute to it, I guess many others would as well!  Please make the application open source.

**Comment by jeppeolesen** [comment]
Thank you for the interest and the offer - open sourcing the entire app isn't on the roadmap in the near future, so I'm going to close this feature request.

If you're interested in contributing and have some cool ideas, you can consider building an integration that plugs into MultiViewer's internal API.

**Reply by d-schotman** [reply]
How would that integration into MultiViewer's internal API work?

**Reply by jeppeolesen** [reply]
@d-schotman When MultiViewer is running, it exposes an API via localhost that you can use to control various aspects of the application, or read data being used for things like live timing.

If you open the application, go to settings and scroll down to the experimental section, you can find the API endpoint and options. If you open the endpoint URL in a browser, you'll be taken to the API playground which include documentation on all the various calls you can make.

**Reply by d-schotman** [reply]
@jeppeolesen awesome! Thanks so much! I'm going to tinker around with this and see if I can come up with something cool!
